server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/CloudConnectionConfig.java

```diff
@@ -15,13 +15,17 @@
  */
 package com.oceanbase.odc.service.connection.model;
 
+import java.util.Map;
+
 public interface CloudConnectionConfig {
     String getClusterName();
 
     String getTenantName();
 
     String getOBTenantName();
 
+    Map<String, String> getProperties();
```

Comment:
what's this used for?

you may take a look at `ConnectionConfig`, there is a field named `attributes`.